src/frontends/paddle/tests/test_models/gen_scripts/generate_multi_tensor_split.py

```diff
@@ -15,20 +15,27 @@ def create_multi_output_model():
     num_splits = 20
     inp_blob_1 = np.random.randn(2, num_splits, 4, 4).astype(np.float32)
 
-    x = fluid.data(name='x', shape=[2, num_splits, 4, 4], dtype='float32')
-    test_layer = fluid.layers.split(x, num_or_sections=num_splits, dim=1)
-
+    if if paddle.__version__ >= '2.0.0':
```

Comment:
```suggestion
    if paddle.__version__ >= '2.0.0':
```
Can you build successfully in your local machine?

Reply:
> Can you build successfully in your local machine?

sorry, I will check again.